dotCMS/src/main/java/com/dotmarketing/factories/WebAssetFactory.java

```diff
@@ -111,7 +110,7 @@ public static AssetType getObject (String value) {
 	private final static int MAX_LIMIT_COUNT = 100;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "MAX_LIMIT_COUNT" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)